content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Virtual IPs and Service Proxies
+content_type: reference
+---
+
+<!-- overview -->
+Every {{< glossary_tooltip term_id="node" text="node" >}} in a Kubernetes
+cluster runs a [kube-proxy](/docs/reference/command-line-tools-reference/kube-proxy/)
+(unless you have deployed your own alternative component in place of `kube-proxy`).
+
+The `kube-proxy` component is responsible for implementing a _virtual IP_
+mechanism for {{< glossary_tooltip term_id="service" text="Services">}}
+of `type` other than
```

Comment:
This is not true, kube-proxy is an implementation detail and is not mandatory, there are other projects that implement the kube-proxy functionality

Reply:
> Every node in a Kubernetes
> cluster runs a kube-proxy
> (unless you have deployed your own alternative component in place of `kube-proxy`).

That looks true to me. Maybe we can reword to make the part in parens more obvious @aojea ?